### PR TITLE
Better way to provision dbaas

### PR DESCRIPTION
**Pull request by gen1us2k**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [ ] Have you followed the guidelines in our Contributing document in setting up the application?
* [ ] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [ ] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [ ] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
* [ ] Have you included the specific version of the application to be installed?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot).
* [ ] Updated the version number of the app if that has changed.

### CODE REVIEW

File: percona-monitoring-and-management/secret.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pmm-secret1
+  namespace: default
+type: Opaque
+data:
+  PMM_ADMIN_PASSWORD: $PMM_PASSWORD
```
Comment (denisok) [inline percona-monitoring-and-management/secret.yml:8]: `PMM_ADMIN_PASSWORD: $PMM_PASSWORD` should be
`PMM_ADMIN_PASSWORD: $PMM_ADMIN_PASSWORD`

you have env:
```
configuration:
  PMM_ADMIN_PASSWORD:
```

**Review by denisok — CHANGES_REQUESTED** [review]

File: percona-monitoring-and-management/uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Delete the PMM
+helm uninstall pmm
```
Comment (denisok) [inline percona-monitoring-and-management/uninstall.sh:4]: so, should we also delete Secret and what about PVC?

File: percona-monitoring-and-management/secret.yml (hunk 1 above)
Comment (gen1us2k) [reply percona-monitoring-and-management/secret.yml:8]: We need to encode to base64 first
```PMM_PASSWORD=$(echo ${PMM_ADMIN_PASSWORD}|base64)
```

File: percona-monitoring-and-management/install.sh [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+# Update your local Helm chart repository cache
 helm repo update
-kubectl create ns pmm
 
-helm install pmm percona/pmm --namespace pmm  --set secret.pmm_password=$PMM_ADMIN_PASSWORD --set service.type=$SERVICE_TYPE --set-string pmmEnv.ENABLE_DBAAS="1"
+# Install the PMM  Helm chart
+helm install pmm percona/pmm --set secret.create=false --set service.type=$SERVICE_TYPE
+
+# Wait for pod to be ready
+kubectl wait pods --for condition=Ready --timeout=120s pmm-0
```
Comment (denisok) [inline percona-monitoring-and-management/install.sh]: https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/k8s_helper.sh#L4

120s might not be enough, our container is 1.2Gb

File: percona-monitoring-and-management/uninstall.sh (hunk 2 above)
Comment (gen1us2k) [reply percona-monitoring-and-management/uninstall.sh:4]: I think that we can cover it with the documentation. I'll delete the secret

File: percona-monitoring-and-management/install.sh (hunk 3 above)
Comment (gen1us2k) [reply percona-monitoring-and-management/install.sh]: Okay. even better

**Review by denisok — APPROVED** [review]

**Review by spron-in — APPROVED** [review]

**Pull request merged**